src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_SIZE = 16;
+    private static final int GROW_FACTOR = 2;
+    private static int size;
+    private int capacity;
```

Comment:
current capacity can be calculated based on size and array length, not need to have separate variables for them
```suggestion
```